#Created by Quincy Fleming
#Failure to compile without include "<stdlib.h>"
#Bug: https://bugs.gentoo.org/927801
--- a/patch.c
+++ b/patch.c
@@ -30,6 +30,7 @@
  */
 #include <errno.h>
 #include <string.h>
+#include <stdlib.h>
 #include <libxml/parser.h>
 #include <libxml/tree.h>
 
--- a/program.c
+++ b/program.c
@@ -31,6 +31,7 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <string.h>
+#include <stdlib.h>
 #include <unistd.h>
 #include <libxml/parser.h>
 #include <libxml/tree.h>
